opal/browser/audio/param_schedule.rb

```diff
@@ -0,0 +1,24 @@
+module Browser; module Audio
+
+  class ParamSchedule
+    include Native
+
+    alias_native :value,               :setValueAtTime
```

Comment:
Do you usually set multiple values for the same time? As in values at the same time for `value`, `linearRamp`, and `exponentialRamp`?

If yes make it `at(time).value(..).linear_ramp(..).exponential_ramp(..)`.